BoundingBox object_bb
geometry_msgs/Point center
float32 maxHeight
---
bool isAvailable
bool isSupported
